packages/roosterjs-plugin-picker/lib/PickerPluginSetSuggestingEvent.ts

```diff
@@ -0,0 +1,5 @@
+export const PICKER_PLUGIN_SET_SUGGESTING_EVENT = 'PICKER_PLUGIN_SET_SUGGESTING_EVENT'
```

Comment:
Curious whether you think it's worthwhile to allow this to be provided in PickerOptions in the future? Is there value in knowing specifically which picker plugin the event source is?

Reply:
that's what the intent behind including the idPrefix as part of the the event was to cover. That way autocomplete plugin can track each of the pickers independently to make sure no pickers are open.